packages/fs-detectors/src/detect-builders.ts

```diff
@@ -531,6 +517,20 @@ function detectFrontBuilder(
     return { src, use: `${use}${withTag}`, config };
   }
 
+  if (
+    pkg &&
+    (framework === undefined || createdAt < Date.parse('2020-03-01'))
+  ) {
+    const deps: PackageJson['dependencies'] = {
+      ...pkg.dependencies,
+      ...pkg.devDependencies,
+    };
+
+    if (deps['next']) {
+      framework = 'nextjs';
+    }
+  }
```

Comment:
Why was this moved?

Reply:
Reverted the change, but I think the code may be flawed. `createdAt` defaults to `0`, so if `framework` is valid, but `createdAt` is `undefined`, it'll force the framework to be `nextjs`.